### PR TITLE
Feat: skip utxo selection

### PR DESCRIPTION
**Pull request by szymonlesisz**
- use requested utxo without coin selection algorithm (use case: coin control feature in suite)
- prioritize utxo using `required` field (use case: rbf)
- replace travis with github actions (execution time reduced by from ~26 min to ~9 min)

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [8.x]
```
Comment (prusnak) [inline .github/workflows/nodejs.yml]: Don't we want something newer, e.g.?
```suggestion
        node-version: [10.x, 12.x, 14.x, 15.x]
```

File: shell.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  })
+{ };
+
+stdenv.mkDerivation {
+  name = "hd-wallet-dev";
+  buildInputs = [
+    autoPatchelfHook
+    git
+    nodejs-12_x
+    (yarn.override { nodejs = nodejs-12_x; })
```
Comment (prusnak) [inline shell.nix]: We don't need to override anymore - nodejs 12.x is default in the commit defined in the shell.nix
```suggestion
    nodejs
    yarn
```

File: src/build-tx/coinselect-lib/inputs/accumulative.js [hunk 3]
```diff
@@ -13,6 +13,36 @@ export default function accumulative(utxos, outputs, feeRate, options) {
     const inputs = [];
     const outAccum = utils.sumOrNaN(outputs);
 
+    // split utxos into required and the rest
+    const requiredUtxo = [];
```
Comment (prusnak) [inline src/build-tx/coinselect-lib/inputs/accumulative.js]: Let's use "s" at the end for consistency.

```suggestion
    const requiredUtxos = [];
```

**Review by prusnak — CHANGES_REQUESTED** [review]
Looks OK, left couple of comments

File: shell.nix (hunk 2 above)
Comment (szymonlesisz) [reply shell.nix]: addressed 94cfe3939703780fc7339da05445871de45fa958

File: src/build-tx/coinselect-lib/inputs/accumulative.js (hunk 3 above)
Comment (szymonlesisz) [reply src/build-tx/coinselect-lib/inputs/accumulative.js]: addressed d6a96bd4d0b8544ffe3e703b47e44a9fe3783e17

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (szymonlesisz) [reply .github/workflows/nodejs.yml]: i've removed 10.x as well, end of life is announced to 2021-04-30

**Review by prusnak — APPROVED** [review]

**Pull request merged**